scripts/check-qstash-env.js

```
@@ -0,0 +1,69 @@
+#!/usr/bin/env node
+
+/**
+ * This script checks if QStash environment variables are properly loaded
+ * Run it with: node scripts/check-qstash-env.js
+ */
+
+// Load environment variables
+require('dotenv').config();
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)